build/styles/about/USC-main.css

```diff
@@ -0,0 +1,298 @@
+html {
+    font-size: 16px;
+}
+
+
+:root {
+    --text-color: #212121;
+    --border-color: #D9D4D7;
+    --secondary-text-color: #44425A;
+    --emphasis: #E1A011;
+}
+
+
+.main {
+    /* justify-content: center; */
+    align-items: center;
+    /* padding: 0 3rem; */
+}
+
+.main__heading {
+    margin-top: 2.75rem;
+    max-width: 635px;
+    color: var(--text-color);
+    text-align: center;
+    font-size: 4rem;
+    font-weight: 600;
+}
+
+.main__container {
+    width: 100%;
+    padding: 4.5rem 0 3.75rem;
+    border-bottom: solid 1px var(--border-color);
+}
+
+.main__container--about-USC {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 2.25rem;
+}
+
+.main__container--about-USC .USC-logo {
+    max-width: 17.5rem;
+    max-height: 17.5rem;
+}
+
+.main__container--text-content {
+    display: flex;
+    flex-direction: column;
+    gap: 13px;
+    color: var(--secondary-text-color);
+}
+
+.main__sub-heading {
+    font-size: 2.25rem;
+    font-weight: 600;
+    color: var(--secondary-text-color);
+}
+
+.main__text {
+    text-align: justify;
+    font-size: 1rem;
+    font-weight: 400;
+}
+
+.main__container--USC-officers {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-bottom: 3.875rem;
+}
+
+.main__container--USC-officers .main__sub-heading {
+    font-size: 2.5rem;
+}
```

Comment:
I think we should put the presentation of officers in a separate CSS file and rename it into something reusable. The HTML of this can also be placed under the components folder.

We'll not do these changes immediately, but I'll take note of it.